src/command_spec.rs

```diff
@@ -65,8 +65,11 @@ impl CommandSpec {
         self
     }
 
-    pub fn add_env(&mut self, key: String, value: String) -> &mut Self {
-        self.envs.push((key, value));
+    pub fn add_env<T>(&mut self, key: T, value: T) -> &mut Self
```

Comment:
FYI: This was deliberate by me. Functions, which offer passing by reference, which immediately clone the value, are considered non-idiomatic by some. I guess we need to agree on a consistent style at some point.

Reply:
I don't have a definitive opinion on this. I would say it is a case-by-case decision. In this case, I oriented myself towards the standard library: https://doc.rust-lang.org/nightly/std/process/struct.Command.html#method.env